Tasks/AzureAppServiceManageV0/package.json

```diff
@@ -20,7 +20,7 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
-    "azure-pipelines-tasks-azure-arm-rest-v2": "^3.223.3",
```

Comment:
do we want to limit to this specific version?

Reply:
Yes. We want to replace it with exactly the same version that is currently used in the task. 
Task owners will be able to change this later